src/main/webapp/dev/js/common/instructorFeedbacks.js

```diff
@@ -94,6 +98,42 @@ function formatResponsesVisibilityGroup() {
     });
 }
 
+/**
+ * Initializes the modal providing more information on different options
+ * for setting visibility of a feedback session
+ */
+function setupSessionVisibilityInfoModal() {
+    const modalText = $(`#${ParamsNames.FEEDBACK_SESSION_SESSIONVISIBILITY}`).html();
```

Comment:
I think you don't need to extract the HTML of the element and then recreate it. You can select the element, then remove it from the current DOM, remove the `display: none;` style, and then pass the element to `showModalAlert`.

Also, you don't need so many functions that do this same thing for the 3 modal. Should abstract to the same function.

Reply:
> Also, you don't need so many functions that do this same thing for the 3 modal. Should abstract to the same function.

I was trying to follow the same pattern as other functions ? They have similar logic. For example, `formatSessionVisibilityGroup` and `formatResponsesVisibilityGroup`, `updateUncommonSettingsSessionVisibilityInfo` and `updateUncommonSettingsEmailSendingInfo`, etc.

Reply:
You are not forbidden to write better code than the current code, aren't you. Anyway, you should move your 3 functions down below `formatSessionVisibilityGroup` so that `formatResponsesVisibilityGroup` and `formatSessionVisibilityGroup` stay next to each other.

Reply:
Sorry I just meant for consistency :) Anyway I'll change it to a common function.